spec/requests/apartments_spec.rb

```diff
@@ -2,6 +2,24 @@
 
 RSpec.describe "Apartments", type: :request do
   describe "GET /index" do
-    pending "add some examples (or delete) #{__FILE__}"
+    it "gets a list of apartments" do
+      Apartment.create(
+          street: '444 Beyonce Blvd.',
+          city: 'Houston',
+          state: 'Texas',
+          manager: 'Tina Knowles',
+          email: 'MsTina@test.com',
+          price: '2000',
+          bedrooms: 3,
+          bathrooms: 2,
+          pets: 'yes',
+          image: 'https://images.unsplash.com/photo-1630699034276-0be879da7ebf?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTF8fGFwYXJ0bWVudCUyMGludGVyaW9yfGVufDB8fDB8fA%3D%3D&auto=format&fit=crop&w=1400&q=60'
+      )
+
+      get '/apartments'
+      apartment = JSON.parse(response.body)
+      expect(response).to have_http_status(200)
```

Comment:
This looks great, perhaps also check to make sure that the length of the array that comes back is 1.